opensafely/check.py

```diff
@@ -14,10 +14,14 @@
 
 
 def add_arguments(parser):
-    pass
+    parser.add_argument(
+        "--error",
+        help="Error (exit 1) if restricted datasets found",
+        action="store_true",
+    )
 
 
-def main():
+def main(error=False):
```

Comment:
I'd be tempted to make the default error=True, and only disable it in the special case of local `run`

Reply:
Agree